gradle/plugins/src/main/kotlin/com.hedera.gradle.spotless.gradle.kts

```diff
@@ -23,16 +23,6 @@ spotless {
     // optional: limit format enforcement to just the files changed by this feature branch
     ratchetFrom("origin/develop")
 
-    format("misc") {
-        // define the files to apply `misc` to
-        target("*.md", ".gitignore")
-
-        // define the steps to apply to those files
-        trimTrailingWhitespace()
-        indentWithSpaces()
-        endWithNewline()
-    }
-
```

Comment:
Should not be removed, but instead should only cover the `.gitignore` and other misc files.

Reply:
Agreed @nathanklick, have re-instated.